src/ui/routes/apis.js

```diff
@@ -294,6 +295,17 @@ router.get('/:api', function (req, res, next) {
                 debug(thisApp);
             }
 
+            let isAppSubscribed = apps.some(ele => ele.mayUnsubscribe && ele.mayUnsubscribe === true);
```

Comment:
@darshan-clvt , does it impact other apis ? i don't see any if condition being used to set this logic only for cortellis api

Reply:
- it wont impact on there apis.  
it will impact for only cortellis api.pug Ui, 
when we use the variable disbaleSubscription Btn has button value property in other api's subscribed application button. 
by default, In pug when we assign property value, disabled property. it was readying false. there we could not disbale buttons ,